### ISSUE TITLE
Outputting the content for dynamically creating a zip

### ISSUE THREAD

**Issue by Astinox** [Chumper/Zipper#97]
Hi there,
I am currently working on a project which requires us to generate dynamic zip files with images and summaries. Is there any way of archieving this without saving the .zip file itself and just getting the contents for a user to download?

**Comment by Chumper** [comment]
I dont think that is possible as this is just a wrapper around the `zip` functionality of php: http://php.net/manual/en/book.zip.php

If there is a way to do it with the extension, then it can be implemented
